### ISSUE TITLE
Permissions error

### ISSUE THREAD

**Issue by JulieVelghe** [solspace/craft3-freeform#1413]
### What happened?

When I log in to my website as a defined moderator user, I get a server error when viewing the submissions, telling me the user doesn't have right permissions.
This goes away when I enable everything for this user, but that is not optimal. I don't want my user to be able to change the freeform settings.

![image](https://github.com/solspace/craft-freeform/assets/46524074/4a7392d6-9619-4f5b-a729-321d6d084ed3)

![image](https://github.com/solspace/craft-freeform/assets/46524074/d21c61dd-e127-4ca5-95d7-43be71de2bab)

### Errors and Stack Trace (if available)

```shell
{name: "Forbidden", message: "User is not permitted to perform this action", code: 0,…}
code
: 
0
error
: 
"User is not permitted to perform this action"
exception
: 
"yii\\web\\ForbiddenHttpException"
file
: 
"C:\\Users\\julie\\Documents\\CabinetC\\Repos\\Ilvo\\vendor\\solspace\\craft-freeform\\packages\\plugin\\src\\Library\\Helpers\\PermissionHelper.php"
line
: 
56
message
: 
"User is not permitted to perform this action"
name
: 
"Forbidden"
status
: 
403
```

### How can we reproduce this?

1. upgrade from a freeform 4 version
2. create a user, give all rights except 'access settings'
3. view the submissions

### Freeform Edition

Pro

### Freeform Version

5.5.2

### Craft Version

4.10.3

### When did this issue start?

After upgrading from older Freeform version

**Comment by kjmartens** [comment]
Sorry about this @JulieVelghe. We're looking into it now.

**Comment by kjmartens** [comment]
This should be resolved in Freeform 5.5.4+ now. 🙂
